### PR TITLE
first app setup

### PR DESCRIPTION
**Pull request by YannickZahinda**
### Blog_App

In this pull request I made sure that postgres database is set up.
I build project schema following requirements

### CODE REVIEW

File: db/migrate/20220716112940_create_posts.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
+      t.integer :comment_counter
+      t.integer :likes_counter
+
+      t.timestamps
+    end
+  end
+end
```
Comment (andersonlebon) [inline db/migrate/20220716112940_create_posts.rb:12]: - [ ] Nice job creating tables however this table should be associated with the user table. Kindly use a migration like this :point_down: 

```.rb
class CreatePosts < ActiveRecord::Migration[7.0]
  def change
    create_table :posts do |t|
      t.string :title
      t.string :text
      t.integer :comment_counter
      t.integer :likes_counter
      t.references :author, foreign_key: { to_table: 'users' }

      t.timestamps
    end
  end
end
```

File: db/migrate/20220716113150_create_comments.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :text
+
+      t.timestamps
+    end
+  end
+end
```
Comment (andersonlebon) [inline db/migrate/20220716113150_create_comments.rb:9]: - [ ] Nice job creating tables however this table should be associated with the user and post tables. Kindly use a migration like this :point_down

```.rb
class CreateComments < ActiveRecord::Migration[7.0]
  def change
    create_table :comments do |t|
      t.string :text
      t.references :post, foreign_key: true
      t.references :author, foreign_key: { to_table: 'users' }

      t.timestamps
    end
  end
end
```

File: db/migrate/20220716113228_create_likes.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes, &:timestamps
+  end
+end
```
Comment (andersonlebon) [inline db/migrate/20220716113228_create_likes.rb:5]: - [ ] Nice job creating tables however this table should be associated with the user and post tables. Kindly use a migration like this :point_down

```.rbclass CreateLikes < ActiveRecord::Migration[7.0]
  def change
    create_table :likes do |t|
      t.references :author, foreign_key: { to_table: 'users' }
      t.references :post, foreign_key: true
      t.timestamps

    end
  end
end
```

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @YannickZahinda 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220716112940_create_posts.rb (hunk 1 above)
Comment (rloterh) [reply db/migrate/20220716112940_create_posts.rb]: This issue is well resolved! ✔️

File: db/migrate/20220716113150_create_comments.rb (hunk 2 above)
Comment (rloterh) [reply db/migrate/20220716113150_create_comments.rb]: This issue is well resolved! ✔️

File: db/migrate/20220716113228_create_likes.rb (hunk 3 above)
Comment (rloterh) [reply db/migrate/20220716113228_create_likes.rb]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Zahinda,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Highlights 👍
- Database schema properly implemented


<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**